test/conformance/context/urContextCreateWithNativeHandle.cpp

```diff
@@ -70,7 +70,7 @@ TEST_P(urContextCreateWithNativeHandleTest, SuccessWithUnOwnedNativeHandle) {
     uint32_t ref_count = 0;
     ASSERT_SUCCESS(urContextGetInfo(ctx, UR_CONTEXT_INFO_REFERENCE_COUNT,
                                     sizeof(uint32_t), &ref_count, nullptr));
-    ASSERT_EQ(ref_count, 2);
+    ASSERT_EQ(ref_count, 1);
```

Comment:
the answer might be just don't check the reference counts at all and leave it all implementation defined

Reply:
I was just thinking what cl does is right but for the native handle itself. But, this query is querying the UR reference count. and UR reference should not be altered whether it owns the native handle or not but rather the reference count of the native handle itself is the one that should be changed. So, it feels more to test this feature should be a backend specific feature rather than a CTS test here. This would be a test for only that creating a UR object from owned/unowned handles is a success.

I don't have strong impression about this still as I think it is still to be more properly defined so, if you think we should wait for this to define `isNativeHandleOwned` more properly, I am okay with that.

Reply:
> the answer might be just don't check the reference counts at all and leave it all implementation defined

I think that make more sense to make this more generic for now.